docs/support-policy.md

```diff
@@ -22,6 +22,16 @@ Currently, these are versions:
 
 There are packages for `arm64` and `amd64` architectures.
 
+## Debian packages
+
+We aim to provide packages for all [stable releases](https://www.debian.org/releases/)
```

Comment:
Ah, I don't think we should really say that, because as mentioned in the issue:
> what versions of Debian are supported (depending on the Ubuntu ones available)

so, we really only aim to support some Ubuntu versions, which incidentally means that some Debian versions end up being supported at all.

So here I think we should say that we _don't_ aim to support all stable debian releases, but only those that are matching the supported Ubuntu versions.